tilda.appdata.xml

```diff
@@ -38,6 +38,5 @@
   </screenshot>
  </screenshots>
  <url type="homepage">https://github.com/lanoxx/tilda</url>
- <updatecontact>lanoxx_at_gmx.net</updatecontact>
- <project_group>Tilda</project_group>
```

Comment:
The _project_group_ tag is invalid according to the [appstream-util](https://people.freedesktop.org/~hughsient/appstream-glib/) command line tool:

`$ appstream-util validate-relax tilda.appdata.xml`
`tilda.appdata.xml: FAILED:`
`? tag-invalid           : <project_group> is not valid`
`Validation of files failed`

(The _updatecontact_ tag is okay according to this tool, but [invalid](https://appstream.debian.org/sid/main/issues/tilda.html) according to the appstream-generator used by Debian/Ubuntu.)